pkg/gatewayserver/io/basicstationlns/basicstationlns.go

```diff
@@ -171,7 +185,31 @@ func (s *srv) handleTraffic(c echo.Context) error {
 		}
 	}()
 
-	// TODO: Start downlink processing in goroutine, see gRPC frontend.
+	// Process downlinks in a separate go routine
+	go func() {
+		for {
+			select {
+			case <-conn.Context().Done():
+				return
+			case down := <-conn.Down():
+				dnmsg := messages.DownlinkMessage{}
+				token := s.correlations.GenerateNextToken()
+				s.correlations.Store(token, down.GetCorrelationIDs())
+				dnmsg.FromDownlinkMessage(ids, *down, token)
+				msg, err := dnmsg.MarshalJSON()
+				if err != nil {
+					logger.WithError(err).Error("Failed to marshal downlink message")
+					continue
+				}
+
+				logger.Info("Send downlink message")
```

Comment:
Info? We'll get hundreds of these per second 😱 

Seeing that it's also happening in other frontends of the GS, we can leave it in, but this may have to change in the future.

Reply:
More like tens per second I think. Even on our current biggest deployments, we have a downlink rate of 10-20 PPS.